packages/calypso-build/webpack.config.js

```diff
@@ -106,10 +112,7 @@ function getWebpackConfig(
 					presets,
 					workerCount,
 				} ),
-				SassConfig.loader( {
-					preserveCssCustomProperties: false,
-					prelude: '@import "~@automattic/calypso-color-schemes/src/shared/colors";',
-				} ),
+				SassConfig.loader( { postCssConfigPath } ),
```

Comment:
The default `postcss.config.js` is now an internal property of the `SassConfig` module. Therefore, I think it would be helpful to move the config to `calypso-build/webpack/sass/postcss.config.js` and load it as default from the `sass.js` module.

The default usage:
```js
// webpack.config.js
const SassConfig = require( '@automattic/calypso-build/webpack/sass' );

loaders = [
  ...,
  SassConfig.loader()
]
```
Then it automatically loads the default config.

Reply:
While you're right that the config is now more tightly coupled to the `SassConfig` module, I'm a bit reluctant to move it into a `webpack/sass/` subdir, since it's neither directly relevant to Webpack, nor to Sass; that just happens to be the location of the module. (Even if we renamed the subdir to something like `style/`, the  `webpack/` part still wouldn't be quite right.)

Another argument: If we did this, we might also feel compelled to move `babel.config.js` into `webpack/transpile/`. I think I'd rather keep the individual tools' config files at `calypso-build`'s root level, since that replicates more closely where they would usually reside in a project (and thus make them easier to find for a user -- it's less intuitive for someone to look for a PostCSS config in `webpack/sass/` or `webpack/style/` or for a Babel config in `webpack/transpile`, respectively; it also makes it harder to tell which tools have default configs in `calypso-build`).

I think I'm overall trying to get away from the Webpack 'presets' concept that I introduced into `calypso-build` at some point; their interface has alway been somewhat of an irky, arbitrary addition on top of the actual tools. It seems that supporting a project's own configs (as this PR does for PostCSS) also removes the need for a lot of them. I think I'm kind of hoping to eventually merge the presets back into `calypso-build`'s `webpack.config.js`. :thinking: